src/compiler/preprocess/index.ts

```diff
@@ -83,6 +83,80 @@ async function replace_async(
 	return out.concat(final_content);
 }
 
+
+
+// import decoded sourcemap from mozilla/source-map/SourceMapGenerator
+// forked from source-map/lib/source-map-generator.js
+// from methods _serializeMappings and toJSON
+function DecodedSourcemapFromSourceMapGenerator(this: any) {
```

Comment:
it's a little unexpected to me that this would start with a capital D unless it's a constructor